spec/features/axe_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Autocomplete suggestions', api: false, js: true do
```

Comment:
It looks like this describe title was copied here and should be changed.  How about "Accessibility testing"?  What about renaming the whole file to be `accessibility_spec.rb` so it is independent of the underlying tech used to do the a11y testing?

Reply:
```suggestion
RSpec.describe 'Accessibility testing', api: false, js: true do
```

Reply:
I thought about that too; my concern was these tests are really about the technology and how and what it chooses to report on (which isn't and can't be a complete picture of the accessibility anyway) 🤷‍♂️ .